Tests/Controller/ExceptionControllerTest.php

```diff
@@ -5,7 +5,7 @@
 use Symfony\Component\DomCrawler\Crawler;
 use Symfony\Bundle\FrameworkBundle\Client;
 
-class DefaultControllerTest extends WebTestCase
```

Comment:
What do you mean? PSR 0 states that the classname should always equal the filename, so this should be checked automatically